src/common/warnings.rs

```diff
@@ -0,0 +1,76 @@
+use crate::common::{FileEntry, Warning};
+use std::ffi::OsString;
+use std::fmt;
+
+/// Wraps warnings to provide more information when printing.
+pub struct Warnings {
```

Comment:
I think this struct should be named `Output`. What do you think?
/cc @dotenv-linter/core

Reply:
`Output` does sound better, but I think it fits more if that were the name for `Vec<Warnings>`. Each `Warnings` is meant to be a file and the errors it contains.